app/views/hyrax/homepage/_how_to_use.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="col-md-2"></div>
```

Comment:
Instead of a dummy div, try `col-md-offset-2` on the first element below
or `col-sm-offset-2` in this case I guess.

Reply:
col-sm-offset-2 used successfully